### ISSUE TITLE
[Accessibility]: Notepad: Relevant action buttons are not focused when activating "Remove all notes" button

### ISSUE THREAD

**Issue by fkling** [sourcegraph/sourcegraph#35168]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34198

### Problem description

1. Enable notepad by going to https://sourcegraph.com/notebooks and click the "Enable Notepad" button (notepad should be visible in the bottom right corner).
2. Perform a search on https://sourcegraph.com/search that yields a non-trivial amount of search results.
3. Add at least one entry to the notepad
4. Focus and active the "Remove all notes" button

![2022-05-09_16-29](https://user-images.githubusercontent.com/179026/167492869-ae2773fd-fe1f-4de8-b435-2713231702dc.png)

At this point the confirmation panel shows up but the focus is still on the "Remote all notes" button, which means the user has to <kbd>Shift</kbd>+<kbd>Tab</kbd> to the "Cancel" and "Yes, delete" buttons (and focusing "over" the "Create Notebook" button in that process).

### Expected behavior

Focus the "Cancel" button when the panel shows up, so that that action can be activated directly, and "Delete all" can be activated after a single tab.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
